contracts/package.json

```diff
@@ -30,8 +31,8 @@
     "@nomiclabs/hardhat-ethers": "^2.0.0",
     "@nomiclabs/hardhat-solhint": "^2.0.0",
     "@nomiclabs/hardhat-waffle": "^2.0.0",
-    "@openzeppelin/hardhat-upgrades": "^1.6.0",
     "@openzeppelin/contracts": "2.5.0",
+    "@openzeppelin/hardhat-upgrades": "^1.6.0",
```

Comment:
haha. I see you re-ordered so that those packages are in alphabetical order. I'm OCD too... so thank you :)